barretenberg/bbup/README.md

```diff
@@ -0,0 +1,35 @@
+# BBup
+
+BBup is a CLI tool that makes it easy to install the [Barretenberg](https://github.com/AztecProtocol/aztec-packages/blob/master/barretenberg/README.md) proving backend.
+
+Although extensible, it assumes you are using [Noir](https://noir-lang.org) as the frontend language.
```

Comment:
What does extensible mean here? As a user I don't know what that means for me, maybe just drop it

Reply:
I meant to hint you could use `barretenberg` with other frontend languages, but that's quite farfetched. I'll just drop it.